Documentation/CodeSnippets/Extbase/Annotation/Multiple.rst.txt

```diff
@@ -5,14 +5,16 @@
    :caption: EXT:blog_example/Classes/Domain/Model/Post.php
 
    use TYPO3\CMS\Extbase\Persistence\ObjectStorage;
+   use TYPO3\CMS\Extbase\Annotation\ORM\Lazy;
+   use TYPO3\CMS\Extbase\Annotation\ORM\Cascade;
    
    class Post extends AbstractEntity
    {
        /**
         * @var ObjectStorage<Comment>
-        * @Extbase\ORM\Lazy
-        * @Extbase\ORM\Cascade("remove")
+        * @Lazy
+        * @Cascade("remove")
         */
-       public ObjectStorage $comments;
+       protected ObjectStorage $comments;
```

Comment:
The property is protected and there is no getter. Does this work?

Reply:
These where just excepts but as the getters and setters don't really add much infomation to the topic I made them public